src/library/Uno.Cupertino/Styles/Application/Colors.xaml

```diff
@@ -83,5 +83,7 @@
 					 Color="{ThemeResource OpaqueSeparatorColor}" />
 	<SolidColorBrush x:Key="CupertinoLinkBrush"
 					 Color="{ThemeResource LinkColor}" />
+	<SolidColorBrush x:Key="RadioButtonBorderBrush"
+					 Color="{ThemeResource RadioButtonBorderColor}" />
```

Comment:
This should be defined in the RadioButton.xaml style file since this is not a native style defined by Cupertino.

Reply:
We leave it as is unless we change the style - as discussed!